BlazorWasmDemo/Server/appsettings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "Origin": "https://fido2-blazorwasm.onrender.com"
```

Comment:
Should probably be https://localhost:7124/

Reply:
Oh yes absolutely, will make another PR.

Reply:
Ah already fixed nvm :)